src/components/Video/Controls/Controls.module.scss

```diff
@@ -9,13 +9,24 @@
   padding-bottom: 24px;
   font-size: 17px;
 
+  &.isFocusMode {
+    position: fixed;
+    width: 100vw;
+    padding-right: 1rem;
```

Comment:
rem is relative to the font-size of the root element (<html>), while em is relative to this element's own font-size.

As we could change the font size of this element at different breakpoints to the root element, I tend to go with the rule of thumb of – rem can be used for font sizes, em is better for layout sizing. Sometimes that doesn't make sense, but most of the time it does.